src/components/SettingsCheckbox/SettingsCheckbox.vue

```diff
@@ -0,0 +1,81 @@
+<!--
+  - @copyright Copyright (c) 2019 Julius Härtl <jus@bitgrid.net>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
```

Comment:
A <docs> section would be nice :)

Reply:
A sorry, my typing skills failed me. I meant a `<docs>` section, so we have an example how to use this component and how it looks like in the documentation. :see_no_evil:

Reply:
See https://github.com/nextcloud/nextcloud-vue/pull/671/files#diff-e00e2c81e3689faed3469a4bbfd5964fR23-R64 for an example